pkg/config_test.go

```diff
@@ -0,0 +1,64 @@
+package pkg
+
+import (
+	"io/ioutil"
+	"os"
+	"os/user"
+	"testing"
+)
+
+func TestGetregistryParams(t *testing.T) {
+	usr, err := user.Current()
+	if err != nil {
+		t.Logf("could not retrieve current user: %v", err)
+		t.SkipNow()
+	}
+	if usr.Uid != "0" {
+		t.Logf("current user does not have UID of zero (got %s) "+
+			"so cannot create registry cred location, skipping", usr.Uid)
+		t.SkipNow()
+	}
```

Comment:
no...here the tests will be skipped only if we don't have permission to create files in the `/var/run/` directory.